packages/stickers/src/index.js

```diff
@@ -16,22 +16,22 @@
 /**
  * Internal dependencies
  */
-import * as beautyStickers from './beauty';
-import * as cookingStickers from './cooking';
-import * as diyStickers from './diy';
-import * as entertainmentStickers from './entertainment';
-import * as fashionStickers from './fashion';
-import * as fitnessStickers from './fitness';
-import * as travelStickers from './travel';
-import * as wellbeingStickers from './wellbeing';
+import * as beautyStickers from './fresh-and-bright';
+import * as cookingStickers from './food-and-stuff';
+import * as doersGetMoreDoneStickers from './doers-get-more-done';
+import * as weeklyEntertainmentStickers from './weekly-entertainment';
+import * as fashionOnTheGoStickers from './fashion-on-the-go';
+import * as noDaysOffStickers from './no-days-off';
+import * as experienceThailandStickers from './experience-thailand';
+import * as wellbeingSleepStickers from './sleep';
 
 export default {
```

Comment:
Related question: how well does this scale once we have 50 templates (presumably all loaded from our CDN)?

cc @littlemilkstudio